charts/nginz/templates/conf/_nginx.conf.tpl

```diff
@@ -193,6 +193,7 @@ http {
 
     zauth_keystore {{ .Values.nginx_conf.zauth_keystore }};
     zauth_acl      {{ .Values.nginx_conf.zauth_acl }};
+    oauth_key      {{ .Values.nginx_conf.oauth_key }};
```

Comment:
Is this a pubkey or a privkey? I guess it's a pubkey, but this should probably be named explicitly…

Reply:
It's a public key. Was following the examples above: `zauth_keystore` is the path to the public zauth key.

Reply:
Yes, zauth has the same naming problem, and we probably can't change that. I'd still prefer if try to keep the new stuff less confusingly named than staying consistent with the bad existing naming.